tests/common/plugins/loganalyzer/loganalyzer.py

```diff
@@ -150,6 +151,24 @@ def _verify_log(self, result):
                     .format(self.expected_matches_target, len(self.expect_regex))
                 raise LogAnalyzerError(err_target + result_str)
 
+    def save_matching_errors(self, result_log_errors):
+        """
+        save all the log errors in a file on the player
+        :param result_log_errors: list of all the errors we found in the log - result["match_messages"].values()
+        """
+        if result_log_errors:
+            log_errors = ''
+            for error_list in result_log_errors:
+                log_errors += ''.join(error_list)
+
+            tmp_folder = "/tmp/loganalyzer/{}".format(self.ansible_host.hostname)
+            os.makedirs(tmp_folder, exist_ok=True)
+            file_path = os.path.join(tmp_folder, "log_error.json")
```

Comment:
The file name should contain the test name. And preferably also a timestamp. Otherwise, each test will override the logs of the previous logs.

Reply:
update the file with the fix